RD_WorkShop2/script.js

```diff
@@ -0,0 +1,95 @@
+document.addEventListener("DOMContentLoaded", () => {
+    const userList = document.getElementById("user-list");
+    const userPostsList = document.getElementById("user-posts");
+    let usersData = null;
+
+    function clearElement(element) {
```

Comment:
function declaration should be outside of `cument.addEventListener("DOMContentLoaded", () => {`